backend/WikiContrib/Install.md

```diff
@@ -6,6 +6,16 @@ The present doc deals about installing and running `backend`. If you go inside `
 
 # Steps to setup server locally
 
+## Using Docker 
+
+1. Install Docker in your system (https://docs.docker.com/install/)
+2. Get to the  Dockerfile directory : `cd backend/WikiContrib`  
+3. Run `docker build .` to build an image.
+4. Run `docker  run -td  -p 8000:8000  <image-id>` to run the container.
```

Comment:
What is image-id here? May be add some more info here and how to obtain it from the results of the previous step.

Reply:
Noted!